src/routes/LoadSafePage/LoadSafePage.tsx

```diff
@@ -64,13 +66,13 @@ function Load(): ReactElement {
     const ownerEntries = ownerList
       .map((owner) => {
         const ownerFieldName = `owner-address-${owner.address}`
-        const ownerNameValue = values[ownerFieldName]
+        const ownerNameValue = values[ownerFieldName] || values[FIELD_SAFE_OWNER_ENS_LIST][owner.address]
         return {
           ...owner,
           name: ownerNameValue,
         }
       })
-      .filter((owner) => !!owner.name)
+      .filter(Boolean)
```

Comment:
This change would add owners to the address book even if no name was set. Prior to it they were filtered out. As per our UI both owners and address book entries are required to have a name. I think for the scope of this task it makes sense to revert to the old filter but we should evaluate if the name needs to stay a required field. Currently we import owners with empty names even though the field is required. cc @iamacook